atoma-proxy/src/server/mod.rs

```diff
@@ -21,6 +21,9 @@ pub type Result<T> = std::result::Result<T, error::AtomaProxyError>;
 /// The max_tokens field in the request payload.
 pub(crate) const MAX_TOKENS: &str = "max_tokens";
```

Comment:
Should we add a deprecation notice here as well?

Reply:
Nice observation, added